test/e2e_test.go

```diff
@@ -118,6 +121,39 @@ func TestMultipleSignatures(t *testing.T) {
 	must(verify(pub2, imgName, true, nil), t)
 }
 
+func TestSignBlob(t *testing.T) {
+
+	var blob = "someblob"
+	td1 := t.TempDir()
+	td2 := t.TempDir()
+	defer func() {
+		os.RemoveAll(td1)
+		os.RemoveAll(td2)
+	}()
+	bp := filepath.Join(td1, blob)
+
+	if err := ioutil.WriteFile(bp, []byte(blob), 0644); err != nil {
+		t.Fatal(err)
+	}
+
+	_, privKeyPath1, pubKeyPath1 := keypair(t, td1)
+	_, _, pubKeyPath2 := keypair(t, td2)
+
+	ctx := context.Background()
+
+	// Verify should fail on a bad input
+	mustErr(cli.VerifyBlobCmd(ctx, pubKeyPath1, "", "badsig", blob), t)
+	mustErr(cli.VerifyBlobCmd(ctx, pubKeyPath2, "", "badsig", blob), t)
+
+	// Now sign the blob with one key
+	sig, err := cli.SignBlobCmd(ctx, privKeyPath1, bp, true, passFunc)
+	if err != nil {
+		t.Fatal(err)
+	}
+	// Now verify should work with that one, but not the other
+	must(cli.VerifyBlobCmd(ctx, pubKeyPath1, "", string(sig), bp), t)
```

Comment:
this might also need

```
mustErr(cli.VerifyBlobCmd(ctx, pubKeyPath2, "", string(sig), bp), t)
```
here to make sure it only works with the first but not the second